docs/scss/_blog.scss

```diff
@@ -0,0 +1,33 @@
+.ov-masthead--home {
```

Comment:
you can also drop all the styles for the homepage since we get those already in the _homepage.scss file.

Reply:
Okay I'll do that. In this case would the only CSS tag for the blog should be the background-image then?

Reply:
yup for now, but you can keep adding blog specific styles in there too.